client/src/components/categorieslist/CategoriesList.jsx

```diff
@@ -60,6 +64,7 @@ CategoriesList.propTypes = {
   category: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
+      replaceAll: PropTypes.func.isRequired,
```

Comment:
something is wrong here

Reply:
i moved this NavLink to categories page map fonction for dellet this proptypes